src/plugins/data/public/ui/query_editor/language_selector.tsx

```diff
@@ -14,12 +14,15 @@ import {
 import { getQueryService } from '../../services';
 import { LanguageConfig } from '../../query';
 import { Query } from '../..';
+import { Dataset } from '../../../common';
+import { darkCssDistFilename } from '../../../../../../packages/osd-ui-shared-deps/index';
 
 export interface QueryLanguageSelectorProps {
   query: Query;
   onSelectLanguage: (newLanguage: string) => void;
   anchorPosition?: PopoverAnchorPosition;
   appName?: string;
+  dataset?: Dataset;
```

Comment:
Doesnt the query object already contain the dataset?

Reply:
+1

Reply:
yes, removed extra prop